### PR TITLE
Upgrade to Kotlin 1.4.

### PR DESCRIPTION
**Pull request by zach-klippenstein**

### CODE REVIEW

**Comment by patrickyin** [comment]
`rc` removed!!!

File: .buildscript/configure-compose.gradle [hunk 1]
```diff
@@ -19,7 +19,7 @@ android {
     compose true
   }
   composeOptions {
-    kotlinCompilerVersion "1.4.0-rc"
+    kotlinCompilerVersion "1.4.0"
```
Comment (patrickyin) [inline .buildscript/configure-compose.gradle:22]: Do we still need this line?

Comment (patrickyin) [reply .buildscript/configure-compose.gradle:22]: Tested. We can remove this.

Comment (zach-klippenstein) [reply .buildscript/configure-compose.gradle]: Yep, but it can just refer directly to `Versions.kotlin` now. Fixed.

**Pull request merged**